src/HasteModuleLoader/HasteModuleLoader.js

```diff
@@ -226,9 +226,12 @@ Loader.prototype._execModule = function(moduleObj) {
     || (onlyCollectFrom && onlyCollectFrom[modulePath] === true);
 
   if (shouldCollectCoverage) {
+    if (!this._CoverageCollector) {
+      this._CoverageCollector = require(this._config.coverageCollector);
+    }
```

Comment:
I move ``require`` here, because previously ``require``-ing is in constructor and I didn't know the ``require`` there is not a real one.